packages/vue/src/components/molecules/SfRange/SfRange.vue

```diff
@@ -0,0 +1,226 @@
+        };
+      },
+    },
+  },
+  data() {
+    return {
+      config: {},
+    };
+  },
+  watch: {
+    min: {
+      handler(value) {
+        if (this.$refs.range) {
+          return this.updateRangeOptions({
+            range: {
+              min: value >= this.max ? this.max - this.step : value,
+              max: this.max,
+            },
+          });
+        }
+      },
+      immediate: true,
+    },
+    max: {
+      handler(value) {
+        return this.updateRangeOptions({
+          range: {
+            min: this.min,
+            max: value <= this.min ? this.min + this.step : value,
+          },
+        });
+      },
+      immediate: true,
+    },
+    step: {
+      handler(value) {
+        return this.updateRangeOptions({
+          step: value,
+        });
+      },
+      immediate: true,
+    },
+    tooltips: {
+      handler(value) {
+        if (value) {
+          value =
+            this.slidersInitialValues.length === 2
+              ? [
+                  wNumb(this.formatTooltipsValues),
+                  wNumb(this.formatTooltipsValues),
+                ]
+              : [wNumb(this.formatTooltipsValues)];
+        } else {
+          return value;
+        }
+        return this.updateRangeOptions({
+          tooltips: value,
+        });
+      },
+      immediate: true,
+    },
+    formatTooltipsValues: {
+      handler(value) {
+        if (this.tooltips)
+          return this.updateRangeOptions({
+            tooltips:
+              this.slidersInitialValues.length === 2
+                ? [
+                    wNumb(this.formatTooltipsValues),
+                    wNumb(this.formatTooltipsValues),
+                  ]
+                : [wNumb(this.formatTooltipsValues)],
+          });
+      },
+      immediate: true,
+    },
+    horizontalOrientation: {
+      handler(value) {
+        let orientationValue = "horizontal";
+        value
+          ? (orientationValue = "horizontal")
+          : (orientationValue = "vertical");
+        return this.resetAndChangeOption({
+          orientation: orientationValue,
+        });
+      },
+      immediate: true,
+    },
+    ltrDirection: {
+      handler(value) {
+        let directionValue = "ltr";
+        value ? (directionValue = "ltr") : (directionValue = "rtl");
+        return this.resetAndChangeOption({ direction: directionValue });
+      },
+      immediate: true,
+    },
+  },
+  mounted() {
+    this.config = {
+      start: this.slidersInitialValues,
+      range: {
+        min: this.min >= this.max ? this.max - 1 : this.min,
+        max: this.max,
+      },
+      step: this.step,
+      connect: true,
+      direction: this.ltrDirection ? "ltr" : "rtl",
+      orientation: this.horizontalOrientation ? "horizontal" : "vertical",
+      behaviour: "tap-drag",
+      tooltips: false,
+      keyboardSupport: true,
+    };
+    this.noUiSliderInit();
+  },
+  beforeDestroy() {
+    this.$refs.range.noUiSlider.destroy();
```

Comment:
if (this.$refs.range && this.$refs.range.noUiSlider)

Reply:
`this.$refs.range?.noUiSlider?.destroy();`